### ISSUE TITLE
actionUpload controller - return filename in case of filename conflict

### ISSUE THREAD

**Issue by piotrpog** [craftcms/cms#6099]
This would be useful in case we want to use `actionUpload` controller on the frontend, without using any custom modules. 

In the case of filename conflict, data returned by controller looks like this:

```
assetId: 622
conflict: "A file with the name “eee.jpg” already exists."
conflictingAssetId: "621"
filename: "eee.jpg"
```

What is lacking, is the filename of duplicate file that was created on the server (which i something like `eee_200518_221543.jpg`). It would be useful if this filename was returned too, so it can be displayed on frontend.

**Comment by andris-sevcenko** [comment]
This has been fixed for Craft 3.4!

**Comment by brandonkelly** [comment]
Craft 3.4.21 is out now with that change.
